po/zh-CN.po

```diff
@@ -1914,7 +1914,7 @@ msgid ""
 "example, using the example on the previous page, make `src/main.rs` look like"
 msgstr ""
 "将 `src/main.rs` 中的样板代码替换为你自己的代码。例如， 使用上一页中的示例，"
-"将 `src/main.rs` 改为："
+"将 `src/main.rs` 改为这样"
```

Comment:
If this change is also caused by the poedit warning, I'd suggest to keep the previous transaltion which would be fit to the context.

Reply:
This is Poedit's original suggestion
<img width="804" alt="image" src="https://github.com/google/comprehensive-rust/assets/11788119/ae15b3ce-0985-4df8-bebb-22cc20940cbd">